pyproject.toml

```diff
@@ -43,7 +43,6 @@ kubernetes = "26.1.*"
 podman = "4.9.*"
 rq-scheduler = "^0.10"
 django-ansible-base = { git = "https://github.com/ansible/django-ansible-base.git", tag = "2024.2.12", extras = [
-    "authentication",
```

Comment:
Don't we need this for jwt?

Reply:
No, jwt auth comes from a different library `ansible_base.jwt_consumer`.